styles/searxng/catppuccin.user.css

```diff
@@ -240,10 +241,19 @@ url-prefix("https://xo.wtf/") {
       @base: #1e1e2e;
       @mantle: #181825;
       @crust: #11111b;
+    };
+  };
+
+  :root.theme-auto {
+    #catppuccin(@lightFlavor);
```

Comment:
you need to pass `accentColor` into here, so it should be
```diff
- #catppuccin(@lightFlavor);
+ #catppuccin(@lightFlavor, @accentColor);
```

Reply:
Why is this necessary?

Reply:
We are just trying to enforce common practices with the userstyles and i missed the other part which is why the theme still works.